openach/templates/boards/_board_table.html

```diff
@@ -1,5 +1,38 @@
 {% load i18n %}
-<table class="table table-hover">
+{% load widget_tweaks %}
+{% load templatehelpers %}
+
+
+
+<div class="row justify-content-between">
+    {% with params=request.GET.urlencode %}
+      <div class="dropdown col-sm-4 col-md-6">
+        <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Sort By
+        </button>
+        <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+          <div><a class="dropdown-item " href="{% relative_url '-pub_date' 'sort' params %}">Recent Boards</a></div>
```

Comment:
When writing captions in template, please wrap them in a `translate` tag: https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#translate-template-tag